xlab/data/importer/iex/importer.py

```diff
@@ -56,3 +55,31 @@ def _make_data_base(self, symbol: str, data_date: datetime.datetime,
         data_entry.timestamp.FromDatetime(data_date)
         data_entry.updated_at.FromDatetime(now)
         return data_entry
+
+    def _get_date_range(self, start_date: datetime.datetime, end_date: datetime.datetime):
+        '''
+        calculate date range by counting no. of days between start and end days
+        '''
+
+        date_range = {
```

Comment:
Are these copied from iex-finance? I think it's worth showing some calculation or comment on why these numbers are chosen, e.g. why 84 for 3 months instead of 89 or 90?